### PR TITLE
Add Majordomo overview post

### PR DESCRIPTION
**Pull request by ppeble**

### CODE REVIEW

File: _posts/2015-12-23-majordomo.markdown [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title:  "Implementing Majordomo"
+author: "Phil Trimble"
+date:   2015-12-23 12:00:00
+categories: technology, majordomo
+---
+
+Like most technology outfits we have work that we would like to perform 'off-cycle'. This just means that we don't want to do the work during a time that the customer is waiting. This is a very normal problem and there are many solutions to it out there in the world, so many in fact that it would take forever to list them all.
```
Comment (jazminschroeder) [inline _posts/2015-12-23-majordomo.markdown]: > This just means that we don't want to do the work during a time that the customer is waiting
> maybe
> We don't want to make the customers wait for a particular task that is not critical for their experience at that moment.. something like that..

**Comment by jazminschroeder** [comment]
This is a great topic for a blog post! some feedback

I found the post a bit hard to follow. The main topic is majordomo but there's a lot of info there that took away my attention(rabbit vs zmq, stormtrooper, ruby vs go, arbiter). 

Maybe just a bit of restructuring around the main topic would help. Answer some questions such as:
- What problem drove you to look at the majordomo protocol(why pub/sub wasn't enough)? 
- What's majordomo? -- you already cover this
- How did you implement majordomo?(our learnings w zmq, early implementation with ruby then go) -- you already cover this
- What are your plans to open source our work?

This is a very personal opinion hopefully doesn't sound like I'm over criticizing take it or leave it.

Happy to see a new blog post!!

**Comment by ppeble** [comment]
@jazminschroeder I made a variety of changes based on your feedback, thank you! Can you take another read and let me know what you think?

**Comment by jazminschroeder** [comment]
:dancer:
